core/src/main/scala/shapeless/generic.scala

```diff
@@ -328,6 +335,10 @@ trait CaseClassMacros extends ReprTypes with CaseClassMacrosVersionSpecifics {
     nameStr.contains("$anon") || nameStr == "<refinement>"
   }
 
+  /**
+   * @return the List of pair of `tpe` case accessor like term's name and final result type of that term.
+   * @see [[isCaseAccessorLike]] for the definition of case accessor "like".
```

Comment:
I find that sentence difficult to parse (non-native speaker here). Maybe something like this:
```suggestion
   * @return a List of name and type pairs for the fields of type `tpe`.
   * @see [[isCaseAccessorLike]] for the definition of what is considered a field.
```